### PR TITLE
Add language metadata best practices from string-meta [I18N-ACTION-1241]

### PR DESCRIPTION
**Pull request by aphillips**
- Add the note text from gpuweb, which is action 1241
- Add the language related best practices from string meta to the "TBD" section provided previously

***
<a href="https://pr-preview.s3.amazonaws.com/aphillips/bp-i18n-specdev/pull/106.html" title="Last updated on Jul 13, 2023, 3:06 PM UTC (61c111d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/bp-i18n-specdev/106/3cef306...aphillips:61c111d.html" title="Last updated on Jul 13, 2023, 3:06 PM UTC (61c111d)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -471,9 +471,81 @@ <h4>Establishing the language of inline runs</h4>
...
+
+<div class="req" id="bp_unicode_tag_chars_nonuse">
+<p class="advisement">Specifications SHOULD NOT use the Unicode "language tag" characters (code points <code>U+E0000</code> to <code>U+E007F</code>) for language identification.</p>
+</div>
+
+<div class="req" id="bp_language_indexing">
+<p class="advisement">Specifications SHOULD recommend the use of language indexing when localizable strings can be supplied in multiple languages for the same value.</p>
```
Comment (xfq) [inline index.html]: I don't quite understand this sentence. Could you provide a simple example?

Comment (aphillips) [reply index.html]: Good callout. See if ec6a09b fixes it?

Comment (xfq) [reply index.html]: Looks good to me. Thanks!

File: index.html [hunk 2]
```diff
@@ -475,26 +475,42 @@ <h3>Identifying the language of strings</h3>
...
 
+<p>The exchange of data on the Web, to the degree possible, should use <a>locale-neutral</a> standardized formats. However, some data on the Web necessarily consists of <a>natural language</a> information intended for display to humans. This <a>natural language</a> information depends on and benefits from the presence of language and direction metadata for proper display. Along with support for Unicode, mechanisms for including and specifying the base direction and the natural language of spans of text are one of the key internationalization considerations when developing new formats and technologies for the Web.</p>
+
 
 <p class="reviewComments"><a href="https://github.com/w3c/i18n-activity/labels/t%3Alang_strings_x" target="_blank">See related review comments.</a></p>
+<aside class="links" id="lang_strings_links">
+<p class="links_title">Useful background and overviews for this section</p>
+<ul>
+<li class="w3"><p class="link"><a href="https://w3c.github.io/string-meta/">Strings on the Web: Language and Direction Metadata</a>.</p>
```
Comment (xfq) [inline index.html]: We should link to the /TR version (same for the links below).

Comment (aphillips) [reply index.html]: Done (globally) for string-meta. Other links possibly need attention, but I'll create a separate issue and PR for those.

**Comment by aphillips** [comment]
Per teleconference, merging this. I will address the stylesheet linking in a separate pull request.

**Pull request merged**